KnockKnock-iOS/Scenes/Feed/FeedWrite/Views/FeedWriteView.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  FeedWriteView.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/03/17.
+//
+
+import UIKit
+
+import Then
+
+class FeedWriteView: UIView {
+
+  // MARK: - Constants
+
+  private enum Metric {
+    static let photoAddButtonWidth = 65.f
+    static let photoAddButtonHeight = 65.f
+    static let photoAddButtonTopMargin = 20.f
+    static let photoAddButtonLeadingMargin = 20.f
+
+    static let tagTopMargin = 25.f
+    static let tagLeadingMargin = 20.f
+    static let tagTrailingMargin = -60.f
+    static let tagHeigth = 40.f
+
+    static let promotionTopMargin = 25.f
+    static let promotionLeadingMargin = 20.f
+    static let promotionTrailingMargin = -60.f
+
+    static let shopTopMargin = 25.f
+    static let shopLeadingMargin = 20.f
+    static let shopTrailingMargin = -60.f
+
+    static let selectButtonHeight = 40.f
+
+    static let selectImageTopMargin = 5.f
+    static let selectImageTrailingMargin = -20.f
+
+    static let seperatorHeight = 1.f
+    static let seperatorWidthMargin = -40.f
+
+    static let contentTextViewTopMargin = 20.f
+    static let contentTextViewLeadingMargin = 20.f
+    static let contentTextViewTrailingMargin = -20.f
+    static let contentTextViewHeight = 300.f
+
+    static let doneButtonHeight = 45.f
+    static let doneButtonBottomMargin = -10.f
+    static let doneButtonleadingMargin = 20.f
+    static let doneButtonTrailingMargin = -20.f
+  }
+
+  // MARK: - UI
+
+  let photoCollectionView = UICollectionView().then {
+    let flowLayout = UICollectionViewFlowLayout.init()
+    flowLayout.scrollDirection = .horizontal
+    $0.translatesAutoresizingMaskIntoConstraints = false
+  }
```

Comment:
```suggestion
let photoCollectionView = UICollectionView(
    frame: .zero,
    collectionViewLayout: UICollectionViewFlowLayout().then {
      $0.scrollDirection = .horizontal
    }
  ).then {
    $0.translatesAutoresizingMaskIntoConstraints = false
  }
```
작성해주신 코드는 flowLayout이 선언만 되어있고 아무 동작도 안할것으로 보이는군요??..

그래서 위와 같이 써주면 제대로 동작하고 좀 더 깔끔하게 보이겠군요